spec/rspec_test_helper_spec.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe "RSpec test helper" do
+  class Klass
```

Comment:
Because of yucky Ruby block scoping, this creates a constant in the top-level namespace that doesn't get cleaned up called `Klass` -- it's why the other spec does a `let(:klass)` -- can we do the same here?